extensions/amp-story-360/0.1/amp-story-360.md

```diff
@@ -0,0 +1,222 @@
+scripts:
+  - >-
+    <script async custom-element="amp-story-360"
+    src="https://cdn.ampproject.org/v0/amp-story-360-0.1.js"></script>
+layouts:
+  - Fill
+  - fixed
+  - fixed height
+  - flex item
+  - nodisplay
+---
+
+<!---
+Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+[tip type="important"]Change image sources to ampproject/amphtml/master!!![/tip]
+
+<figure class="centered-fig">
+  <amp-anim alt="amp-story-player example" width="300" height="533" layout="fixed" src="https://raw.githubusercontent.com/processprocess/amphtml/360-docs/extensions/amp-story-360/img/amp-story-360.gif">
+    <noscript>
+    <img alt="amp-story-player example" src="https://raw.githubusercontent.com/processprocess/amphtml/360-docs/extensions/amp-story-360/img/amp-story-360.gif" />
+  </noscript>
+  </amp-anim>
+</figure>
+
+The `amp-story-360` component provides a way to embed 360 images and videos in [Web Stories](https://amp.dev/documentation/guides-and-tutorials/start/create_successful_stories/?format=stories).
+
+## Usage
+
+Use the `amp-story-360` component to render 360 images and video. The component can animate between two points or be explorable via the device's gyroscope sensor. For best results, only use one element per [`amp-story-page`](https://amp.dev/documentation/components/amp-story-page/?format=stories).
+
+### Media Requirements
+
+The `amp-story-360` component renders videos and images in [equirectangular projection](https://en.wikipedia.org/wiki/Equirectangular_projection).
+This is the default format generated by many 360 cameras and 3D rendering software.
+
+<amp-img src="https://raw.githubusercontent.com/processprocess/amphtml/360-docs/extensions/amp-story-360/img/SeanDoran-Quela-sol1462-edited_ver2-sm_600-300.jpg" layout="intrinsic" width="400" height="230">
+
+Credit: [NASA / JPL / MSSS / Seán Doran](https://informal.jpl.nasa.gov/museum/360-video).
+
+### Dimensions
+
+Use a 2:1 aspect ratio for the best distribution of resolution.
+
+### Size
+
+2048 x 1024 (2k) is recommended for images and video.
+
+### Configuration
+
+The `amp-story-360` component accepts 1 descendant. This descendant must be an [amp-img](https://amp.dev/documentation/components/amp-img/?format=stories) or [amp-video](https://amp.dev/documentation/components/amp-video/?format=stories).
+
+#### amp-image
+
+[sourcecode:html]
+<amp-story-360 layout="fill">
+<amp-image layout="fill" src="assets/image360.jpg"></amp-image>
```

Comment:
```suggestion
<amp-img layout="fill" src="assets/image360.jpg"></amp-img>
```

Can you run these snippets through validator.ampproject.org and make sure we didn't forget anything?

Reply:
PR #30832 and #31028 aren't reflected on validator.ampproject.org yet. Once they are these should pass validation.

Reply:
PR #30832 and #31028 aren't reflected on validator.ampproject.org yet. 
Once they are these should pass validation.

Reply:
Those PRs are now on validator.ampproject.org and these are passing :)